intake/catalog/entry.py

```diff
@@ -102,6 +106,7 @@ def plots(self):
 
     def _ipython_display_(self):
         """Display the entry as a rich object in an IPython session."""
+        from IPython.display import display
         contents = self.describe()
         display({  # noqa: F821
```

Comment:
With this import the lint directive is no longer relevant. I had slid on the import because this function is only meant to be invoked in an `IPython` context, when the `display` function is automatically in the global namespace.

Reply:
Right, I see what it was doing now.

Reply:
No harm in importing though, it's more explicit.